### PR TITLE
Update README.adoc

### LINKED ISSUE #17: Terminolgy edits/updates
per  #267 in docs repo

### PR DESCRIPTION
**Pull request by t-koman**
per #267 of docs repo and closes #17

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -17,24 +17,24 @@ permalink: /guides/app-logging/
...
+**EFK (Elasticsearch, FluentD, and Kibana)** is a log aggregation tool, and this guide describes the process of deploying EFK using the ansible-playbook script provided by Kubernetes development (The Origin Community Distribution of Kubernetes).
 
-Use the ansible-playbook that resides in the https://github.com/openshift/openshift-ansible/tree/release-3.11[openshift-ansible] repository to install EFK stacks on an OKD cluster. Use this preconfigured EFK stack to aggregate all container logs. After a successful installation, the EFK deployments reside inside the *openshift-logging* namespace of the OKD cluster.
+Use the ansible-playbook located in the https://github.com/openshift/openshift-ansible/tree/release-3.11[openshift-ansible] repository to install EFK stacks on a Kubernetes development cluster. Then use this preconfigured EFK stack to aggregate all container logs. After a successful installation, the EFK deployments reside inside the *openshift-logging* namespace of the Kubernetes development cluster.
```
Comment (fwji) [inline README.adoc]: I don't think we can replace OKD here with Kubernetes. The ansible script is only for a OKD cluster.

File: README.adoc [hunk 2]
```diff
@@ -17,24 +17,24 @@ permalink: /guides/app-logging/
...
 
-One choice for application logging with log aggregation, based on open source, is **EFK (Elasticsearch, FluentD, and Kibana)**. This guide describes the process of deploying EFK using the ansible-playbook script provided by OKD (The Origin Community Distribution of Kubernetes).
+**EFK (Elasticsearch, FluentD, and Kibana)** is a log aggregation tool, and this guide describes the process of deploying EFK using the ansible-playbook script provided by Kubernetes development (The Origin Community Distribution of Kubernetes).
```
Comment (fwji) [inline README.adoc]: OKD is the acronym for (The Origin Community Distribution of Kubernetes), so cannot be simply replaced by Kubernetes

File: README.adoc [hunk 3]
```diff
@@ -17,24 +17,24 @@ permalink: /guides/app-logging/
...
-* It is much easier to find application logs in Kibana, as logs are not intermixed with verbose Kubernetes logs. 
+* Ease of finding application logs in Kibana, where logs are not intermixed with verbose Kubernetes logs. 
 
-* It provides more flexibility for memory allocation, because users can independently assign system memory to each of the EFK deployments. 
+* Flexible memory allocation, because users can independently assign system memory to each of the EFK deployments. 
 
 == Install openshift-logging 
 
-Installing EFK on OKD requires an https://docs.okd.io/3.11/install/configuring_inventory_file.html[inventory host file] and the openshift-ansible playbook for logging the installation. After you create the inventory file, add the following Ansible variables to the **[OSEv3:vars]** section of the inventory file.
+Installing EFK on Kubernetes development requires an https://docs.okd.io/3.11/install/configuring_inventory_file.html[inventory host file] and the openshift-ansible playbook for logging the installation. First create the inventory file, then add the following Ansible variables to the **[OSEv3:vars]** section of the inventory file.
```
Comment (fwji) [inline README.adoc]: similarly, I would use OKD here as the playbook is OKD exclusive

File: README.adoc [hunk 4]
```diff
@@ -45,22 +45,22 @@ openshift_logging_es_ops_memory_limit=5G
 openshift_logging_es_memory_limit=3G
 ----
 
-There are many other Ansible variables provided by OKD for advanced fine-tuning of the EFK stack. Detailed information about the installation and configuration can be found in OKD documentation https://docs.okd.io/3.11/install_config/aggregate_logging.html[Aggregating Container Logs]. 
+Numerous Ansible variables are available to fine tune the EFK stack. Detailed information about the installation and configuration can be found in Kubernetes development documentation https://docs.okd.io/3.11/install_config/aggregate_logging.html[Aggregating Container Logs].
```
Comment (fwji) [inline README.adoc]: Kubernetes -> OKD

File: README.adoc [hunk 5]
```diff
@@ -45,22 +45,22 @@ openshift_logging_es_ops_memory_limit=5G
...
-* `openshift_logging_es_memory_limit` and `openshift_logging_es_ops_memory_limit` are self-explanatory and can be set according to your preference. For stable operation, allocate at least 2 GB of memory for each of the Elasticsearch deployments. If memory is not specified explicitly, OKD allocates 16 GB of memory for each of the Elasticsearch deployments by default. It is highly recommended to install openshift-logging on systems with at least 32 GB of RAM when `openshift_logging_use_ops` is set to true.
+* `openshift_logging_es_memory_limit` and `openshift_logging_es_ops_memory_limit` are self-explanatory and can be set according to your preference. For stable operation, allocate at least 2 GB of memory for each of the Elasticsearch deployments. If memory is not specified explicitly, Kubernetes development allocates 16 GB of memory for each of the Elasticsearch deployments by default. When `openshift_logging_use_ops` is set to true, install openshift-logging on systems with at least 32 GB of RAM.
```
Comment (fwji) [inline README.adoc]: Kubernetes development -> OKD

File: README.adoc [hunk 6]
```diff
@@ -45,22 +45,22 @@ openshift_logging_es_ops_memory_limit=5G
...
+* `openshift_logging_es_memory_limit` and `openshift_logging_es_ops_memory_limit` are self-explanatory and can be set according to your preference. For stable operation, allocate at least 2 GB of memory for each of the Elasticsearch deployments. If memory is not specified explicitly, Kubernetes development allocates 16 GB of memory for each of the Elasticsearch deployments by default. When `openshift_logging_use_ops` is set to true, install openshift-logging on systems with at least 32 GB of RAM. 
 
-After all the variables are set in the inventory file, run the ansible-playbook command to install the EFK stack onto its current OKD Cluster.
+After all the variables are set in the inventory file, run the ansible-playbook command to install the EFK stack onto its current Kubernetes development cluster.
```
Comment (fwji) [inline README.adoc]: Kubernetes development -> OKD

**Comment by t-koman** [comment]
Hello,
I have signed the CLA.
thanks,
Tom

IBM - WebSphere Application Server Information DevelopmentBuilding 707, Floor 1, Office H612455 South Road, Poughkeepsie, NY 12601T/L: 8-295-8668 PHONE: (845) 435-8668tkoman@us.ibm.com
 
 
----- Original message -----From: CLA assistant <notifications@github.com>To: "kabanero-io/guide-app-logging-okd-3.11" <guide-app-logging-okd-3.11@noreply.github.com>Cc: t-koman <tkoman@us.ibm.com>, Author <author@noreply.github.com>Subject: [EXTERNAL] Re: [kabanero-io/guide-app-logging-okd-3.11] Update README.adoc (#18)Date: Wed, Feb 12, 2020 7:33 PM 
Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our Contributor License Agreement before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us recheck it.

**Comment by t-koman** [comment]
@fwji Frank - please review my terminology update.  Your review is pending...... thanks

**Review by fwji — CHANGES_REQUESTED** [review]

**Comment by fwji** [comment]
Hi @t-koman, please let me know if there are any concerns for using OKD in the guide. A lot of the material communicated in this guide is OKD exclusive, and the content does not apply to Kubernetes.

**Comment by jantley-ibm** [comment]
@fwji @t-koman has made updates per your last review, if you agree with the changes please approve the review so that we can merge -- thanks

**Review by fwji — APPROVED** [review]

**Pull request merged**